src/alias/azext_alias/alias.py

```diff
@@ -210,14 +215,15 @@ def get_full_alias(self, query):
         """
         if query in self.alias_table.sections():
             return query
+
         return next((section for section in self.alias_table.sections() if section.split()[0] == query), '')
 
     def load_full_command_table(self):
         """
         Perform a full load of the command table to get all the reserved command words.
         """
         load_cmd_tbl_func = self.kwargs.get('load_cmd_tbl_func', None)
-        if load_cmd_tbl_func:
+        if load_cmd_tbl_func is not None:
```

Comment:
Between this an previous version, the change is subtle but substantial. Does it mean there is a change that `load_cmd_tbl_func` will be assigned to a value whose _truthiness_ is `False` but `not None`? And that is a positive case?